service/history/workflowTaskHandler.go

```diff
@@ -746,6 +763,16 @@ func (handler *workflowTaskHandlerImpl) handleCommandStartChildWorkflow(
 		return err
 	}
 
+	failWorkflow, err = handler.sizeLimitChecker.failWorkflowIfSearchAttributesSizeExceedsLimit(
+		attr.GetSearchAttributes(),
+		parentNamespace,
```

Comment:
should this be child namespace? or simple do not validate and let start workflow execution fail?